File: Models/AccountDto.cs
namespace test_sf_git_prop.Models;

public sealed class AccountDto
{
    public string Id { get; set; } = string.Empty;
    public string? Name { get; set; }
    public string? AccountNumber { get; set; }
    public string? Phone { get; set; }
    public string? Website { get; set; }
    public string? Industry { get; set; }
    public string? BillingStreet { get; set; }
    public string? BillingCity { get; set; }
    public string? BillingState { get; set; }
    public string? BillingPostalCode { get; set; }
    public string? BillingCountry { get; set; }
}

File: src/main/java/com/ai2dev/testsfgitpropjava/model/AccountDto.java
package com.ai2dev.testsfgitpropjava.model;

import jakarta.validation.constraints.NotBlank;

public record AccountDto(
    String id,
    @NotBlank String name,
    String type,
    String industry,
    String phone,
    String website,
    String email,
    String billingStreet,
    String billingCity,
    String billingState,
    String billingPostalCode,
    String billingCountry
)
{
}
